pkg/kubelet/lifecycle/interfaces.go

```diff
@@ -27,6 +27,23 @@ type PodAdmitAttributes struct {
 	OtherPods []*v1.Pod
 }
 
+const (
+	AppArmorReason                            = "AppArmor"
+	InvalidNodeInfoReason                     = "InvalidNodeInfo"
+	PodOSSelectorNodeLabelDoesNotMatchReason  = "PodOSSelectorNodeLabelDoesNotMatch"
+	PodOSNotSupportedReason                   = "PodOSNotSupported"
+	InitContainerRestartPolicyForbiddenReason = "InitContainerRestartPolicyForbidden"
+	UnexpectedAdmissionErrorReason            = "UnexpectedAdmissionError"
+	UnknownReason                             = "UnknownReason"
+	UnexpectedPredicateFailureTypeReason      = "UnexpectedPredicateFailureType"
+	NodeShutdownNotAdmittedReason             = "NodeShutdown"
+	ForbiddenReason                           = "SysctlForbidden"
+	EvictedReason                             = "Evicted"
+	TopologyAffinityReason                    = "TopologyAffinityError"
+	PredicateFailureErrorReasonTemplate       = "%s"
```

Comment:
In order to be compatible with the previous logic, this method is used here (although it looks strange) 🤔